sdk/storage/azure-storage-perf/src/main/java/com/azure/storage/App.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage;
+
+import com.azure.perf.test.core.PerfStressProgram;
+import com.azure.storage.blob.perf.*;
```

Comment:
Should we undo this * import? Not sure if this matters cause its just the perf package,